### PR TITLE
Update README.md | Add translation section

### PR DESCRIPTION
**Pull request by Oymate**

### CODE REVIEW

**Comment by a1012112796** [comment]
@Oymate Please merge https://github.com/Oymate/gitea/pull/1, Thanks.

File: README.md [hunk 1]
```diff
@@ -101,6 +101,11 @@ NOTES:
 1. **YOU MUST READ THE [CONTRIBUTORS GUIDE](CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST.**
 2. If you have found a vulnerability in the project, please write privately to **security@gitea.io**. Thanks!
 
+## Translating
+
+Translations are done through Crowdin. If you want to translate to a new language ask to add a new language there.
```
Comment (6543) [inline README.md]: For new languages you have to contact the maintainer.

You can also just create an issue for language adding or ask on discord on the #translation section

Comment (6543) [reply README.md]: So it is not explicit over crowdin.

If you need context or find some translation issues, this is of cours over crowdin. Or via Discord

For general translation questions there is a section in the docs. Currently a bit empty but we hope fo fill it as questions pop up.
https://docs.gitea.io/en-us/translation-guidelines/

Comment (6543) [reply README.md]: @Oymate thanks for making a nice sentenc out of my first comment, could you do it for the second one too?

**Review by zeripath — APPROVED** [review]

**Comment by 6543** [comment]
-> https://github.com/go-gitea/gitea/pull/15830#discussion_r630073098

**Comment by Oymate** [comment]
@6543 done!

**Review by a1012112796 — APPROVED** [review]
LGTM

**Comment by 6543** [comment]
@Oymate thanks!

**Pull request merged**